boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -194,3 +198,14 @@
 		};
 	};
 };
+
+#ifdef CONFIG_NETWORKING
```

Comment:
The board should enable ethernet by default, so please remove #ifdef CONFIG_NETWORKING

Reply:
Maybe a better way to say it would be: regardless of whether networking is enabled or not, the eth hardware is there (routed to Eth socket, etc.). If CONFIG_NETWORKING is not enabled on Zephyr's side, it (hw) just won't be used. But there's no need to heavily #ifdef the DTS itself. Same applies to PTP.

(That's just my guess, as I'm learning DTS as we go with all these recent patches. So, if you could ack/nak it, @galak, it would be helpful.)

Reply:
ack, if networking is disabled in Zephyr from a SW perspective, the hardware will not doing anything so there isn't a reason to protect it in the dts this way.